assets/src/scripts/components/PageHistory/PageHistory.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import PageHistoryDiff from './PageHistoryDiff.jsx';
+import PageHistoryList from './PageHistoryList.jsx';
```

Comment:
Can drop the `.jsx` here if you add it to the supported extensions in your webpack config. Also, prefer `.js` over `.jsx` generally.